### ISSUE TITLE
[bug] New version 0.8.1 exits with error

### ISSUE THREAD

**Issue by trueChazza** [talklittle/ffmpex#32]
Hi, thank you for this library - it's been great to use.

Since upgrading to version `0.8.1` the library now exits with this error:

<img width="977" alt="Screen Shot 2021-11-16 at 4 26 59 PM" src="https://user-images.githubusercontent.com/20368025/141890684-593afde7-e8a1-46bd-9e58-ad87daf858c6.png">

I've tried with the documented examples, which also produces this error.
Thanks for looking into this.

**Comment by trueChazza** [comment]
I can confirm downgrading to version `0.7.2` works fine.

**Comment by talklittle** [comment]
What OS?

What happens when you git clone ffmpex and run `mix test`?

**Comment by trueChazza** [comment]
https://hub.docker.com/_/elixir

I run this in a Docker container (with ffmpex as a dependency) and the host being MacOS.

**Comment by trueChazza** [comment]
Cloning the repo and running `mix test` has tests failing:

<img width="792" alt="Screen Shot 2021-11-16 at 5 44 40 PM" src="https://user-images.githubusercontent.com/20368025/141922595-b5c45fcf-77e1-4434-b1b8-a69533639475.png">

Looks to be a permissions issue on my end?

**Comment by trueChazza** [comment]
https://github.com/saleyn/erlexec/blob/master/src/exec.erl#L19-L21

This may help from the upstream provider.

**Comment by talklittle** [comment]
Thanks for the additional info. Looks like erlexec imposes some restrictions running as root. ffmpex will have to check for that situation and work around the restriction.

**Comment by talklittle** [comment]
Released ffmpex v0.8.2 which ought to fix running as root.

**Comment by trueChazza** [comment]
Thank you for your great work!

I updated to version `0.8.2` but experienced the same error. Different error message though, but same 
root permission reference:

<img width="1256" alt="Screen Shot 2021-11-16 at 8 13 35 PM" src="https://user-images.githubusercontent.com/20368025/141947447-768eab27-78e5-4b92-9a71-212436d0b64d.png">

https://elixirforum.com/t/external-processes-with-environment-variables/7859/7

As referenced in the link, once I ran the docker container not as root, version `0.8.2` works flawlessly. Thank you!
